java/clients/pom.xml

```diff
@@ -121,6 +126,12 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>sleeper</groupId>
+            <artifactId>ingest-core</artifactId>
+            <version>0.13.0-SNAPSHOT</version>
+            <scope>compile</scope>
```

Comment:
This should probably be `<version>${project.parent.version}</version>`, and the scope can be omitted.